src/detection/detection-card/use-style.js

```diff
@@ -0,0 +1,13 @@
+import { useMediaQuery, useTheme } from '@material-ui/core'
+
+import { useDesktopStyle } from 'detection/detection-card/desktop-style'
+import { useMobileStyle } from 'detection/detection-card/mobile-style'
+
+export const useStyle = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125618630-2a4df839-0448-4288-b064-87d3569183a4.png)
Make up your mind. Either we name all our styling files `style` or we name them all `use-style`.

Reply:
done